.Noam/Report/01-Creating-a-Basic-Report.md

```diff
@@ -0,0 +1,4 @@
+﻿# Creating a Basic Report
```

Comment:
Virginie - there is no need to specify the title - it is created automatically by the name of the file.

Please fix